src/ONYXKEYS.js

```diff
@@ -193,4 +193,7 @@ export default {
 
     // Validating Email?
     USER_SIGN_UP: 'userSignUp',
+
+    // Store default drawer status
+    DEFAULT_DRAWER_STATUS: 'defaultDrawerStatus',
```

Comment:
**thought:** unsure whether we need to store this in Onyx or not... seems like something that could just be local to the navigation lib.

Reply:
The reason why we use Onyx is because we need a place that can keep the value even after refresh the page.

Reply:
But why do we need to keep the value after a refresh of the page? 

What is the expected behavior if someone directly navigates to a `/r/<reportID>` route?  Would we show the report or the sidebar?

@parasharrajat do you know the expected behavior and can you point me to where we decided how it should work?

Reply:
I asked this https://github.com/Expensify/App/issues/8126#issuecomment-1069403962 with Expected output https://github.com/Expensify/App/issues/8126#issuecomment-1069452614. 
It should show the report screen. 

But the main issue was that If the user is viewing LHN, refreshing it should land back on LHN and vice-versa.

Reply:
Yeah I guess my question could be re-phrased as... do we need this feature to track the drawer state at all?

What is the issue with this logic?

- When `/` -> Show drawer open
- When `/r/123` -> Show report page (drawer closed)

?

Switching to the sidebar and refreshing the page e.g. will remember that the drawer is open when we are on the report page. Does this mean in other contexts when I want to navigate _directly to a report page_ it will instead show me the sidebar if it was last set as `'open'`? That sounds undesirable.

Reply:
Yes, you're right.

I have another approach for that case, but it's not 100% working as expected. There is a case if we paste the same URL as the current URL it will read as `reload` type, other than that can be returned as `navigate` type.

https://developer.mozilla.org/en-US/docs/Web/API/PerformanceNavigationTiming/type
```
        if (_.map(window.performance.getEntriesByType('navigation'), nav => nav.type).includes('navigate')) {
            return path === '/' ? CONST.DRAWER_STATUS.OPEN : CONST.DRAWER_STATUS.CLOSED;
        }

        return defaultDrawerStatus;
```

Reply:
Ah sorry, if I'm understanding the proposal... you are saying that we should differentiate between a "reload" and a "navigate" - but I'm not sure why. 

Seems like determining the default state of the drawer is only necessary when a new app window opens or is reloaded.

Overall, it feels like the drawer status is something that the linkingConfig in `react-navigation` should handle and be based not on history - but on the url.

I'm going to bring this to the Slack channel just to get some clarification.